static/src/javascripts/projects/common/modules/experiments/tests/membership-engagement-banner-tests.js

```diff
@@ -81,6 +81,14 @@ define([
     };
 
     return [
+        new EditionTest('UK', 'MembershipEngagementBannerUkTest13', '2016-11-24', '2017-1-5', 'gdnwb_copts_mem_banner_aubanner__')
```

Comment:
Yep, good spot, thank you! Damn, it would be better if those were auto-generated...